app/pages/organizations/organization-page.jsx

```diff
@@ -183,11 +193,39 @@ class OrganizationPage extends React.Component {
             </div>
             <div className="organization-researcher-words">
               <h4 className="organization-details__heading"><Translate content="organization.home.links" /></h4>
-              <ul>
-                <li className="organization-details__link">Blog</li>
-                <li className="organization-details__link"><i className="fa fa-facebook fa-fw" />- @orgFacebook</li>
-                <li className="organization-details__link"><i className="fa fa-twitter fa-fw" />- @orgTwitter</li>
-              </ul>
+              <div className="organization-details__links">
+                {!!rearrangedLinks.length && rearrangedLinks.map((link, i) => {
+                  if (link.path) {
+                    let iconForLabel;
+                    if (Object.keys(SocialIcons).indexOf(link.site) >= 0) {
+                      iconForLabel = SocialIcons[link.site];
+                    } else {
+                      iconForLabel = 'globe';
+                    }
+                    return (
+                      <a
+                        key={i}
+                        className="organization-details__link organization-details__link--social"
+                        href={`${link.url}`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        <i className={`fa fa-${iconForLabel} fa-fw fa-2x`} />
+                        <span> - @{link.path}</span>
+                      </a>);
+                  }
+                  return (
+                    <a
+                      key={i}
+                      className="organization-details__link organization-details__link--external"
+                      href={link.url}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      {link.label}
```

Comment:
Design question - have we considered adding a link icon for the external links (maybe, eg, [`fa-link`](http://fontawesome.io/icon/link/)? Having the social icons but nothing for the links looks a little uneven to me, but not a blocker if you like the links without an icon.

Reply:
good question, I think I like it as is where external links don't have an icon, but defer to @beckyrother's opinion.

Reply:
That's a great point James, thanks. It would be great to give an indication that the link is leaving the site – let's do icons. We've used [this one](http://fontawesome.io/icon/external-link/) elsewhere.

Reply:
I hate to open up the link behavior can of worms, but we should be consistent across the site on link behavior. Do project external links open in a new tab? Should any link open in a new tab? We also don't enforce any particular way links are opened that are submitted in markdown from projects (and in fact let them choose with a kind of markdown).

Reply:
currently project sub-nav links open in a new tab, though [the related code](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/project-page.cjsx#L168) looks strange to me, not sure exactly why the links open in a new tab.

Reply:
OK, as long as this is consistent with projects, though if the icon is added, it should probably be added there too. 

The project external link is using a named target, so it opens in a new tab, and when clicked again will open in the same tab rather than a new one.

Reply:
Ah a named tab, that's a good way to do it, I'll do the same here unless there's a reason not to.